nikola/plugin_categories.py

```diff
@@ -80,6 +82,18 @@ def inject_dependency(self, target, dependency):
         self.site.injected_deps[target].append(dependency)
 
 
+class PostScanner(BasePlugin):
+    """The scan method of these plugins is called by Nikola.scan_posts."""
+
+    def scan(self):
+        """Creates a list of posts from some source. Returns timeline, global_data:
+
+        timeline is a list of Post objects
+        global_data is a dictionary where keys are source paths and values are posts.
```

Comment:
Is `global_data` still in use? It seems that neither the `scan_posts` plugin uses it, nor does `Nikola.scan_posts` expects it.